commands/deploy.go

```diff
@@ -67,7 +74,8 @@ var deployCmd = &cobra.Command{
                   [--constraint PLACEMENT_CONSTRAINT ...]
                   [--regex "REGEX"]
                   [--filter "WILDCARD"]
-				  [--secret "SECRET_NAME"]`,
+				  [--secret "SECRET_NAME"]
```

Comment:
Can you fix the indentation while you're changing this, `--replace` and `--update` also need fixed above this.